en-US/snippets/2017/may2017_b.lang

```diff
@@ -9,6 +9,15 @@
 # LIMIT: 180 characters (will be used on desktop). Please rephrase as needed to fit.
 # Desktop snippet
 # Links to Google Play
+# Replace YOURLANGUAGE with Arabic, Farsi, Hebrew or Urdu, the language you are localizing in.
+;Firefox for Android now supports YOURLANGUAGE. Hooray!” <a>Get it now.</a>
```

Comment:
There's a stray double quote here after !

Once you've fixed that, update also other locales and we can merge